packages/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -23,6 +23,7 @@ interface IInMeetingActions {
   canRaiseHand?: boolean;
   canLowerAllHands?: boolean;
   canLowerSomeoneElsesHand?: boolean;
+  canEndMeeting?: boolean;
```

Comment:
issue: This is slightly misleading, because a host CAN end a meeting, even though the display hints will not be provided (they are only provided when there are multiple participants), so the `canEndMeeting` value will be `false`.

I think the name of this should reflect the fact that the display hint says the UI can show both the leave meeting and the end meeting options. Something like `leaveAndEndMeetingAvailable` perhaps (I'm open to better suggestions!)?